integrations/newrelic/main.py

```diff
@@ -0,0 +1,3 @@
+# noinspection PyUnresolvedReferences
+# ruff: noqa: F401
+from newrelic_integration import ocean
```

Comment:
Your ocean file is small so lets keep it simple and put it here

Reply:
it becomes longer, prefer to leave it there